design/recovery-internals.md

```diff
@@ -85,19 +85,27 @@ Master interface is stored in `serverDBInfo`. Once the CC recruits the master, i
 
 *How does each role, such as tLog and data distributor (DD), register its interface to master and CC?*
 
-* tLog monitors `serverDBInfo` change and sends its interface to the new master; 
+* tLog monitors `serverDBInfo` change and sends its interface to the new master;
 
-* Data distributor (DD) and ratekeeper rejoin themselves to CC because they are no longer a part of the recovery process;
+* Data distributor (DD) and Ratekeeper rejoin themselves to CC because they are no longer a part of the recovery process (they are moved out of the master process since 6.2 release);
```

Comment:
Change it to 
`They have been moved out of the master process since 6.2 release, before which they are part of the master recovery in the FDB recovery procedure`.